lib/portlayer/network/context.go

```diff
@@ -1412,6 +1412,71 @@ func (c *Context) deleteScope(s *Scope) {
 	delete(c.scopes, s.Name())
 }
 
+func (c *Context) UpdateContainerNameInScope(h *exec.Handle) error {
```

Comment:
Per discussion, please remove stop/start containers in the rename test case.

Reply:
Sure, will do.